### PR TITLE
media-sound/erec: fix LICENSE, use HTTPS, EAPI=7 bump

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

EAPI=7 bump for media-sound/erec + using https, fixing LICENSE

### CODE REVIEW

File: media-sound/erec/erec-2.2.0.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+src_compile() {
+	emake CC="$(tc-getCC)" CXX="$(tc-getCXX)" \
+		CXXFLAGS="${CXXFLAGS}" CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" -j1
+}
+
+src_install() {
+	dobin erec
+	dodoc README
```
Comment (a17r) [inline media-sound/erec/erec-2.2.0.1-r1.ebuild]: drop it

File: media-sound/erec/erec-2.2.0.1-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	docinto html
```
Comment (a17r) [inline media-sound/erec/erec-2.2.0.1-r1.ebuild]: `local HTML_DOCS=( README.html )`

File: media-sound/erec/erec-2.2.0.1-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	dodoc README.html
```
Comment (a17r) [inline media-sound/erec/erec-2.2.0.1-r1.ebuild]: `einstalldocs`

**Comment by mm1ke** [comment]
@a17r 
Actually, i already tried this. However, in order not to install ```README.html``` twice (because by default ```einstalldocs``` would install all ```README*``` files) i would need to set the ```DOCS``` variable as well. 

This means i wouldn't save any lines here, but i can do this if that is preferred :)

**Comment by a17r** [comment]
Yes, it would be preferred ;)

**Comment by mm1ke** [comment]
done :)

**Pull request closed without merging**